js/controllers/form/DOIPubIdExportFormHandler.js

```diff
@@ -51,14 +51,12 @@
 	$.pkp.controllers.form.DOIPubIdExportFormHandler.
 			prototype.submitAction_ = function(submitButton, event) {
 
-		var $formElement = this.getHtmlElement(),
-				idPattern = new RegExp('(.*)-button-'),
-				button = event.target.id,
-				idPatternResult = idPattern.exec(button),
+		var idPattern = new RegExp('(.*)-button-'),
```

Comment:
I wonder if this will open us up to some kind of strange XSS attack. It might be safer to use `[a-z0-9]+` instead of `.*`